mailer.js

```diff
@@ -311,11 +299,11 @@ module.exports = function (log) {
       subject: gettext('Your Firefox Account password has been changed'),
       template: 'passwordChangedEmail',
       templateValues: {
+        passwordChangeLinkAttributes: this._initiatePasswordChange(message.email),
```

Comment:
That's a follow on PR, @ryanfeeley pointed out the same thing, so I opened #155.